lib/Parse/ParseDeclCXX.cpp

```diff
@@ -1655,6 +1655,14 @@ void Parser::ParseClassSpecifier(tok::TokenKind TagTokKind,
     ParseForanySpecifier(DS);
   }
 
+  if (Tok.is(tok::kw__Itype_for_any)) {
+    // TODO(abeln): add error handling here
```

Comment:
Please remove your GitHub name from the comment.

Reply:
Done